index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  
+  <body>
+    <nav id="navbar">
+    <a href="#welcome-sectionn" class="nav-link">section thing</a>
+    <a href="#projectss" class="nav-link">section thing</a>
+    <a href="#About" class="nav-link">section thing</a>
+    </nav>
+    <br>
+    <br>
+    <section id="welcome-sectionn" class="welcome-sectionn">
+      <h1 class= "h1">Hello World!</h1>
+      <p  class= "p1">Hey We are Zermarden and Shaheer  and Joel
+        we are web developer</p>
+    </section>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <section id="projectss" class="projectss">
+      <div id="welcome-section">
+      <h1>SHAHEER & ZERMARDEN</h1>
+      <h1>OUR PORTFOLIO</h1>
+    </div>
+    </section>
+    <section id="3rdsection" class="3rdsection">
+    <div id="projects">
+      <div>
+        <h1>Shaheer's Profile on GitHub</h1>
+        <a href = "https://github.com/ShaheerCH"
+        id="profile-link"
+        target = "_blank"
+        class = "project-tile">Link to GitHub</a>
+        <h3>Zermarden's Profile on github</h3>
+        <a id="profile-link" href = "https://github.com/Nedramrez"
+        target = "_blank"
+        class = "project-tile">Link to GitHub</a>
+      </div>
+    </div>
+  </section>
+    <div id="About">
+      About us
+    </div>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
I kindly suggest you not to leave commented pieces of code, because it  is regarded as a bad practice.